generic_petg_175.xml.fdm_material

```diff
@@ -59,11 +59,28 @@ Generic PETG 1.75mm profile. The data in this file may not be correct for your s
             <machine_identifier manufacturer="Creality3D" product="creality_ender3" />
             <machine_identifier manufacturer="Creality3D" product="creality_ender4" />
             <machine_identifier manufacturer="Creality3D" product="creality_ender5" />
-            <setting key="print cooling">50</setting>
+            <setting key="print cooling">100</setting>
             <setting key="standby temperature">200</setting>
-            <setting key="retraction speed">45</setting>
+            <setting key="retraction speed">35</setting>
             <setting key="retraction amount">5</setting>
             <setting key="print temperature">240</setting>
+            <setting key="heated bed temperature">80</setting>
```

Comment:
This changes settings for the Creality printers. I don't think you intended to do that.

Reply:
I didn't mean to ping the PR. I hit the wrong button in the GitHub app while poking around. Totally an accident.

If I made changes to the Creality Profiles, it was unintentional.

Reply:
I've reverted the changes to the Creality profiles but merged your other changes. They should've been merged together with your PR to Cura, but we forgot.